iOS/Projects/Features/Record/Sources/Presentation/WorkoutEnvironmentScene/ViewController/WorkoutEnvironmentSetupViewController.swift

```diff
@@ -115,11 +115,8 @@ private extension WorkoutEnvironmentSetupViewController {
 
   func bindStartButton() {
     workoutPeerSelectViewController
-      .startButton
-      .publisher(.touchUpInside)
-      .sink { [weak self] _ in
-        self?.didTapStartButton.send(())
-      }
+      .startButtonDidTapPublisher()
+      .bind(to: didTapStartButton)
       .store(in: &cancellables)
```

Comment:
p3: 만약 함수를 별도로 만드는 것이 좋아보이지 않는다면 아래처럼 해도 상관 없을 것 같아용

```suggestion
    workoutPeerSelectViewController.startButton
      .publisher(.touchUpInside)
      .map { _ in }
      .bind(to: didTapStartButton)
      .store(in: &cancellables)
```

Reply:
buttonEnable도 함수로 빼서, 이것 또한 함수로 빼봤습니다. ㅎㅎ 😊 
조언 감사합니다!